prometheus_speedtest/prometheus_speedtest.py

```diff
@@ -42,6 +44,8 @@ def test(self):
         logging.info('Performing Speedtest')
         client = speedtest.Speedtest(source_address=self._source_address,
                                      timeout=self._timeout)
+        if self._server > 0:
+            client.get_servers(servers=[self._server])
         client.get_best_server()
```

Comment:
[get_best_server](https://github.com/sivel/speedtest-cli/blob/master/speedtest.py#L1427) takes a servers argument as a list.

Why don't we make the servers flag a list and pass that through to `get_best_server`? You'll be able to specify more than one and we won't need the conditional on line 47-- if we pass an empty list, it'll treat behave just as it does today.

Reply:
Good idea, done in https://github.com/jeanralphaviles/prometheus_speedtest/pull/21/commits/cf213e0fb9afdeed0fdaa1e573f9cfb7355e50ce